recipes/spam/meta.yaml

```diff
@@ -0,0 +1,98 @@
+{% set name = "spam" %}
+{% set version = "0.6.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gricad-gitlab.univ-grenoble-alpes.fr/ttk/spam/-/archive/62a7c325dbea0d90314ba19a0ac76a61a23691f4/spam-62a7c325dbea0d90314ba19a0ac76a61a23691f4.tar
+  sha256: bc1c898392eac220784ae534550e2b3b0c071053f9aca870264cc79718b8886e
+  path: ../../  # [win]
+  path: ../     # [win]
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python       # [osx]
+    - python       # [linux]
+    - python =3.7  # [win]
+    - numpy
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - {{ compiler('cxx') }}
+    - pybind11
+    - eigen
+    - cgal-cpp
+    - gmp                                    # [osx]
+    - gmp                                    # [linux]
+    - m2w64-gmp                              # [win]
+    - pyamg
+  host:
+    - python       # [osx]
```

Comment:
you can use `[unix]` to catch both linux and osx.

Reply:
Done